wal/wal.go

```diff
@@ -223,6 +230,30 @@ func Create(lg *zap.Logger, dirpath string, metadata []byte) (*WAL, error) {
 	return w, nil
 }
 
+func (w *WAL) cleanupWAL(lg *zap.Logger) {
```

Comment:
Also, wondering, how about adding a test in the https://github.com/etcd-io/etcd/blob/master/wal/wal_test.go

Reply:
That would be a good idea, however I'm not sure how to reproduce the case that I encountered (a failing fsync on a writable directory). Are there examples of platform specific tests?

Reply:
@spzala Any comments on this?

Reply:
we just need to test if this func works correctly. basically just the rename part.

Reply:
Yes, something similar to the ones in the `wral_test.go` that are created for different wral functions.